README.md

```diff
@@ -48,15 +48,15 @@ Static resources *(aka assets)* are prevalent in web projects. Yet one could say
          - `@<package-scope>/<package-name>@<package-version>`; or
          - `<package-namespace>/<package-name>@<package-version>`.
 
-       When in the case of a scoped or namespaced package, the package directory is composed of two folders.
+       In the case of a scoped or namespaced package, the package directory is composed of two folders.
 
-    4. The organization of assets within the package directory is free; there can be subfolders up to the asset files;
+    4. The organization of assets within the package directory is free; there can be subfolders containing the asset files;
```

Comment:
There is a notion of nested folders that seems to be lost in translation here, don't you think @NathalieNX?